buf.gen.yaml

```diff
@@ -4,6 +4,6 @@ managed:
   go_package_prefix:
     default: github.com/bufbuild/connect-grpchealth-go/internal/gen/go
 plugins:
-  - name: go
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
I'm not sure about this part of the change. This repo already depends on the protobuf runtime - why not continue to keep the plugin and runtime at the same version?

Reply:
Fair enough. Fixed.